lib/web_console/testing/erb_precompiler.rb

```diff
@@ -0,0 +1,22 @@
+require 'web_console/testing/fake_middleware'
+
+module WebConsole::Testing
+  # This class is to pre-compile 'templates/*.erb'.
+  class ERBPrecompiler
+    def initialize(path)
+      @erb  = ERB.new(File.read(path))
+      @view = FakeMiddleware.new(
+        view_path: WebConsole.gem_root('lib/web_console/templates'),
```

Comment:
I'd prefer `gem_root` to just return a `Pathname` object. If you need to join on it, you can call `#join`.